sdk/python/kfp/v2/v2_namespace_alias_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=import-outside-toplevel,unused-import,import-error
+import json
+import os
+import tempfile
+import unittest
+
+
+class V2NamespaceAliasTest(unittest.TestCase):
+    """Test that imports of both modules and objects are aliased
+    (e.g. all import path variants work).
+    """
+
+    def test_1_import_module(self):  # pylint: disable=no-self-use
+        from kfp.v2 import compiler
+        from kfp.v2 import dsl
+
+        @dsl.component
+        def hello_world(text: str) -> str:
+            """Hello world component."""
+            print(text)
+            return text
+
+        @dsl.pipeline(name='hello-world', description='A simple intro pipeline')
+        def pipeline_hello_world(text: str = 'hi there'):
+            """Hello world pipeline."""
+
+            hello_world(text=text)
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            # you can e.g. create a file here:
+            temp_filepath = os.path.join(tempdir, 'hello_world_pipeline.json')
+            compiler.Compiler().compile(
+                pipeline_func=pipeline_hello_world, package_path=temp_filepath)
+
+            with open(temp_filepath, "r") as f:
+                ir_template = json.load(f)
+
+        self.assertIsInstance(ir_template, dict)
```

Comment:
I guess if `json.load(f)` didn't raise any error, this must be true? So seems unnecessary check?
On the other hand, can we assert the v2 aliasing warning was throw?